### ISSUE TITLE
[JOSS-Review][Non-Blocking] Consider open-ai-key and hf-token as optional

### ISSUE THREAD

**Issue by Manvi-Agrawal** [souradipp76/ReadMeReady#17]
Consider making open-ai-key and hf-token as optional for running readme-ready cli, since these variables are useful only for openAI models; and the main objective of this tool is to to support open-source models since OpenAI api is costly.

**Comment by souradipp76** [comment]
@Manvi-Agrawal  This issue has been addressed. Can this issue be marked as closed?

**Comment by Manvi-Agrawal** [comment]
Yes @souradipp76, this issue can be closed. I think you tagged me in issue involving Performance testing, I plan to get back to it by end of this week. Thanks.
